### ISSUE TITLE
[Feature request] Tabs opened from a (compact) pinned tab should appear at the bottom

### ISSUE THREAD

**Issue by sabi0** [eoger/tabcenter-redux#271]
Currently new tabs opened from a (compact) pinned tab appear at the very top of the tab sidebar.
IMO it is not very consistent behavior as usually the top tabs are the "oldest".

I understand the reason for this (new tabs are opened right under the "parent").
But I believe pinned tabs should be treated differently. And their "children" should be added to the bottom of the tab list.

I.e. when I receive a link in a pinned IM tab I would expect the click on the link to behave the same way as: copy link, open a new tab (Ctrl+T), paste and go.

**Comment by eoger** [comment]
Unless you are talking about the "New Tab" button on the sidebar, I can't override Firefox new tab behavior.

**Comment by sabi0** [comment]
No, it's not about the "New Tab" button.
When I (Ctrl+)click a link in a compact pinned tab I would expect a new tab to appear at the bottom of the sidebar.

And since I can drag tabs in the sidebar to re-arrange them I presume the extension has control over the order. So it could make sure that a new tab opened from a pinned tab will always appear at the bottom.
Am I wrong?

It could be as simple heuristic as "if a new tab was added at the very beginning, right after the pinned tabs - move it to the end automatically".

**Comment by eoger** [comment]
Yup no, can't change that (and even if it was possible we'd still follow the Firefox behavior).

**Comment by sabi0** [comment]
Tree Style Tab has a workaround in the latest version.
When a link is clicked in a pinned tab a new "parent" tab is created at the bottom of the sidebar ("Tabs from <pinned tab title>"). And then a new tab is added to this parent.
It's not an ideal solution either. But at least the new tabs appear at the bottom.

When all tabs are closed such "virtual" parent tab is removed automatically.

**Comment by sabi0** [comment]
Actually that extra parent tab can be disabled in the settings. And the new tabs would just appear at the end of the list.

**Comment by Keith94** [comment]
@sabi0 Workaround here: https://addons.mozilla.org/addon/pinned-tabs-fix/

**Comment by sabi0** [comment]
@Keith94 , Thanks. But I don't quite like the idea to fix add-on issue with yet another add-on.
If only Tree Style Tab allowed to disable folding of the nodes I'd just go with it.

**Comment by sabi0** [comment]
@Keith94,
The workaround with the extra addon does work. But it still causes annoying tab bar scrolling (I have a LOT of open tabs). It first scrolls to the top. Then the "fix" kicks in and the tab bar scrolls back to the very bottom.

I guess having the fix integrated in Tab Center Redux would have helped to avoid this useless back and forth scrolling.
